### ISSUE TITLE
Update content - missing required request body parameters 

### ISSUE THREAD

**Issue by eddiegroves** [MrRefactoring/confluence.js#6]
Hi again

`updateContent` is missing required body parameters  - it is currently only [sending 'body'](https://github.com/MrRefactoring/confluence.js/blob/00a3ca987fd2ed1c052aa4ae55f1f0c02c7f6781/src/api/content.ts#L338
)

According to the [docs](https://developer.atlassian.com/cloud/confluence/rest/api-group-content/#api-wiki-rest-api-content-id-put) it requires:

- version
- title
- type

**Comment by MrRefactoring** [comment]
Thanks for report. I will try to fix it tomorrow

**Comment by MrRefactoring** [comment]
Small misunderstanding here. My vision how to use this:

{
  body: {
    version: blablabla,
    title: blablabla,
    body: something
  }
}

**Comment by MrRefactoring** [comment]
But totally agree it’s not clear

**Comment by eddiegroves** [comment]
Compare `createContent`

```
data: {
        ...parameters,
        status: undefined,
      },
```

vs `updateContent`

```
data: parameters.body
```

createContent is working because it's destructing parameters. They both have very similar interfaces for `parameters`.

**Comment by MrRefactoring** [comment]
OMG, I shouldn't have tried to deal with the problem at night :D. You are right about everything, there is indeed a bug, I will try to fix it soon

**Comment by MrRefactoring** [comment]
Fixed in `v1.1.2`. Thanks for contribution!

**Comment by eddiegroves** [comment]
Thanks for the quick replies and fix, awesome work.
